chapters/theoretical_foundations/sections/non-eudlidean-spaces/subsections/transformations.tex

```diff
@@ -0,0 +1,138 @@
+        q_x                                  & q_y                                  & q_z                                  & q_w
+    \end{bmatrix}
+$$
+It can be seen that the translation is an isometry since the row vectors of the matrix are orthonormal.
+
+\subsubsection{Rotation}
+It can be shown that a rotation about an axis through the origin is the same as the Euclidean rotation about the same axis \cite{Philips-Mark-Gunn1992}.
+
+\subsubsection{Camera transformation}
+% The camera transformation is defined in terms of the \textit{camera position} $e$, \textit{gaze direction} $g$, and the \textit{view-up vector} $t$.
+% The camera position is a location that the camera "sees from", the gaze direction is a vector in the direction the camera is looking, and the view-up vector is a vector that points "to the sky".
+% Using these vectors we can define the \textit{view space}, i.e. the camera's local coordinate system with the camera at the geometry origin and basis vectors $i'$, $j'$, and $k'$ defined as follows:
+% \begin{equation*}
+%     \begin{split}
+%         k' = -\frac{g}{\lVert g \rVert}                    \\
+%         i' = \frac{t \times k'}{\lVert t \times k' \rVert} \\
+%         j' = k' \times i'
+%     \end{split}
+% \end{equation*}
+
+The camera transformation allows us to describe the scene from the viewer's point of view.
+The transformation is defined in terms of the \textit{eye position} $e$, and three orthonormal vectors in the tangent space of the eye:
+\begin{enumerate}
+    \item the right direction $i'$,
+    \item the up direction $j'$, and
+    \item the negative view direction $k'$.
+\end{enumerate}
+An example in \autoref{fig:tangent-space} shows the tangent space of the eye, with the $e$ vector marked green, $-k'$ marked blue, and $i'$ marked orange.\\
+\begin{figure}[h]
+    \centering
+    \includegraphics[width=0.4\textwidth]{chapters/theoretical_foundations/sections/non-eudlidean-spaces/resources/tangent-space.png}
+    \caption{Tangent space of the camera}
+    \label{fig:tangent-space}
+\end{figure}
+The transformation can be described by the matrix
+\begin{equation*}
+    V =
+    \begin{bmatrix}
+        i'_x            & j'_x            & k'_x            & \mathcal{L}e_x \\
+        i'_y            & j'_y            & k'_y            & \mathcal{L}e_y \\
+        i'_z            & j'_z            & k'_z            & \mathcal{L}e_z \\
+        \mathcal{L}i'_w & \mathcal{L}j'_w & \mathcal{L}l'_w & e_w
+    \end{bmatrix}
+\end{equation*}
+As the result of the transformation, the eye position is mapped to the geometry origin $g$.
+Furthermore, the vectors $i'$, $j'$, and $k'$ are mapped to $i$, $j$, and $k$, respectively.
+
+\subsubsection{Perspective transformation}
+According to \cite{Szirmay-Kalos2022}, the perspective transformation is given by the matrix \textcolor{red}{[but we don't do it this way, in fact, I don't know how our projection matrix works.]}
```

Comment:
I think we should either show our projection matrix or update ours to match theirs

Reply:
the second option is better

Reply:
I'm going with the 1st option, cuz their matrix aint working